julia_src/reopt_model.jl

```diff
@@ -1410,6 +1611,67 @@ function reopt_results(m, p, r::Dict)
 	return r
 end
 
+function add_re_emissions_results(m, p, r::Dict)
+
+	r["pwf_om"] = round(value(p.pwf_om),digits=4)
+
+	# renewable elec
+	r["annual_re_elec_kwh"] = round(value(m[:AnnualREEleckWh]), digits=2)
+	m[:AnnualREElecPercent] = @expression(m, m[:AnnualREEleckWh]/(sum(p.ElecLoad[ts] for ts in p.TimeStep)))
+	r["annual_re_elec_percent"] = round(value(m[:AnnualREElecPercent]), digits=6)
+
+	# renewable heat
+	if !isempty(p.HeatingTechs)
+		add_re_heat_calcs(m,p)
+		r["annual_re_heat_mmbtu"] = round(value(m[:AnnualREHeatMMBTU]), digits=2)
+		r["annual_heat_mmbtu"] = round(value(m[:AnnualHeatMMBTU]), digits=2)
+		r["annual_re_heat_percent"] = round(value(m[:AnnualREHeatMMBTU])/value(m[:AnnualHeatMMBTU]),digits=6)
+	else
+		r["annual_re_heat_mmbtu"] = 0.0
+		r["annual_re_heat_percent"] = 0.0
+	end
+
+	# BAU preprocessed emissions (used for emissions reduction target calculations; output for testing purposes)
+	r["preprocessed_BAU_year_one_emissions_tCO2"] = round(value(p.BAUYr1Emissions_CO2)/2204.62,digits=2)
```

Comment:
Same thing as above about the lbs to tonne conversion

Reply:
Addressed in [b56776c](https://github.com/NREL/REopt_Lite_API/pull/274/commits/b56776c7dcb718f901681711500d7dd4307f13e0). Thanks!